examples/ShowPlayer.cpp

```diff
@@ -81,13 +86,13 @@ int ShowPlayer::Playback(unsigned int iterations,
   m_loop_delay = delay;
   m_start = start;
   m_stop = stop;
-  m_status = ola::EXIT_OK;
+  m_status = ola::EXIT_SOFTWARE;
 
   if (!m_simulate) {
     ola::io::SelectServer *ss = m_client.GetSelectServer();
     if (duration != 0) {
       ss->RegisterSingleTimeout(
-          duration * 1000,
+          duration,
```

Comment:
As mentioned, this isn't good for backwards compatibility, and on a potentially key option. I can't really think of a good solution though, replacing the argument seems quite clunky.

Reply:
When do BC breaks usually happen in this project?  I've no strong feelings about this particular change, but if a 0.11 (for instance) release is already on the roadmap this feels like a good combination of feature+cleanup to add to that.

Having said that, this is an example program and by no means justifies a new BC breaking release on its own.

Reply:
We try hard to avoid them, as we get so much pain from upstream when we have to deal with their breaking changes. For example #1651 and #1641 and #1422 and a whole pile of libusb cruft.

I'm going to put the rest of my comments about this change in the other discussion on it.